src/main/java/nextstep/subway/exception/LineException.java

```diff
@@ -0,0 +1,7 @@
+package nextstep.subway.exception;
+
+public class LineException extends RuntimeException{
```

Comment:
Exeption도 따로 구성해주셨네요 👍 

```suggestion
public class LineException extends RuntimeException {
```
가끔씩 이런 부분이 보이는데, 이런 부분은 코드 템플릿에서 잡아주지 않나요...?

Reply:
이 부분은 제가 자꾸 까먹는거 같습니다ㅠ
코드 템플릿으로 잡아준다는게 자동으로 잡아주는 설정이있나요??
저는 항상 코드작성한다음에 ctrl+alt+L (윈도우) 눌러서 제가 일일이 해주고있습니다..ㅠㅠ

Reply:
아하..! 작성이 끝나시고, 전체 프로젝트에 대해서도 Reformat Code를 해줄 수 있는데 해당 방법으로 해주셔도 될 것 같아요..!